### ISSUE TITLE
HANKO Key?

### ISSUE THREAD

**Issue by mamscience** [mfts/papermark#219]
Hi

Just stumbled upon the following error when building the app:

```
Error: Please set HANKO_API_KEY and NEXT_PUBLIC_HANKO_TENANT_ID in your .env.local file.
```
Deploying to Vercel results in this fatal error.

No documentation about this... has something been changed?

BR

**Comment by ashrafchowdury** [comment]
Hay, @mamscience,  I've raised a PR about it, check that out here: https://github.com/mfts/papermark/pull/218

**Comment by mfts** [comment]
closed by #218
